PagingWithNetworkSample/lib/build.gradle

```diff
@@ -23,6 +23,7 @@ android {
     }
     kotlinOptions {
         jvmTarget = "1.8"
+        freeCompilerArgs += ["-Xuse-experimental=kotlin.Experimental"]
```

Comment:
is this still needed? we shouldn't expose this anymore

Reply:
Need to finish the upgrade to 1.3.4 first, need to get back to this.

Reply:
Ah - it's here for `flatMapLatest` which we, which reminds me we need to use `collectLatest` (also needs experimental). Updated.